# Receives a TurtleData with the information of the turtle bot that arrived and returns nothing

int32 id
float32 x_pos
float32 y_pos
string color
---

